examples/mysql-query-column-mapping/initialize.bal

```diff
@@ -0,0 +1,24 @@
+import ballerina/sql;
+import ballerinax/mysql;
+
+// Initializes the database as a prerequisite to the example.
+function initialize() returns sql:Error? {
+    mysql:Client mysqlClient = check new (user = "root", password = "Test@123");
+
+    // Creates a database.
+    _ = check mysqlClient->execute(`CREATE DATABASE CUSTOMER`);
+
+    // Creates a table in the database.
+    _ = check mysqlClient->execute(`CREATE TABLE CUSTOMER.Customers
+            (customer_id INTEGER NOT NULL AUTO_INCREMENT, first_name
+            VARCHAR(300), last_name  VARCHAR(300),
+            PRIMARY KEY (customer_id))`);
+
+    // Adds the records to the newly-created table.
+    _ = check mysqlClient->execute(`INSERT INTO CUSTOMER.Customers
+            (first_name, last_name) VALUES ('Peter','Stuart')`);
+    _ = check mysqlClient->execute(`INSERT INTO CUSTOMER.Customers
+            (first_name, last_name) VALUES ('Dan', 'Brown')`);
+
+    check mysqlClient.close();
+}
```

Comment:
```suggestion
}

```

Reply:
Fixed in [2c12211](https://github.com/ballerina-platform/ballerina-distribution/pull/3019/commits/2c122113ca286d8157e77209c8fb0f03811542c0)